packages/wallet-sdk/src/provider/CoinbaseWalletProvider.ts

```diff
@@ -187,8 +187,8 @@ export class CoinbaseWalletProvider extends EventEmitter implements Web3Provider
         this.updateProviderInfo(jsonRpcUrl, Number(_chainId));
       }
 
-      if (event.data.data.action === 'addressChanged') {
```

Comment:
Would we want to maintain both for backwards compat?

Reply:
older dapps that haven't upgraded to the new SDK would be fine because we are still emitting the 'addressChanged' event from extension.

So bascially:
- extension sends both 'addressChanged' and 'addressedChanged'
- if on old SDK, listen to 'addressChanged'
- if on new SDK, listen to 'addressesChanged'

Reply:
One issue though, we will probably release the SDK before the extension with those changes is rolled out 100% to prod

Reply:
We can wait on this PR being release